### PR TITLE
Add coq-mathcomp-algebra-tactics 0.2.0

### PR DESCRIPTION
**Pull request by pi8027**

### CODE REVIEW

File: released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Their instance resolution is done through canonical structure inference.
+Therefore, they work with abstract rings and do not require `Add Ring` and
+`Add Field` commands. Another key feature of this library is that they
+automatically push down ring morphisms and additive functions to leaves of
+ring/field expressions before normalization to the Horner form."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.13" & < "8.15~")}
```
Comment (palmskog) [inline released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam:22]:
```suggestion
  "coq" {(>= "8.13" & < "8.16~")}
```
The package looks good, but maybe you want to allow Coq 8.15+rc1 as well @pi8027? If it turns out there is some incompatibility, we will find out through the opam bench later and can make a change to disallow 8.15.

Comment (pi8027) [reply released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam:22]: IIUC, once we have a release of Coq-Elpi compatible with Coq 8.15, we could widen this constraint. cc: @gares

**Comment by gares** [comment]
See https://github.com/coq/opam-coq-archive/pull/1986

File: released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam (hunk 1 above)
Comment (gares) [reply released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam:22]: please relax the bound

**Pull request merged**